src/utils/logic-app/templateUtils.ts

```diff
@@ -111,9 +128,15 @@ export function generateTemplateParameterDefinition(name: string) {
 }
 
 export function generateTemplateResource(workflow: Workflow) {
-    const { definition, location, name, parameters } = workflow;
+    const { definition, location, name } = workflow;
     const nameParameter = normalizeParameterName(`workflows_${name!}_name`);
 
+    let parameters =  Object.assign({}, workflow.definition.parameters);
+
+    for (var key in parameters) {
```

Comment:
````ts
for (const key of Object.keys(parameters)) {
    parameters[key] = {
        value: `[parameters('workflows_${workflow.name}_parameters_${key}')]`
    };
}
````

Reply:
You might also need to escape any single quotemarks in `key`.

Reply:
Since `key` is coming from parameters already in the definition, therefore it must not have any invalid chars, no?